lib/email/sender.rb

```diff
@@ -239,6 +241,33 @@ def self.host_for(base_url)
 
     private
 
+    def add_attachments(post)
+      email_size = 0
+      max_email_size = 20.megabytes
+      post.uploads.each do |upload|
+        next if FileHelper.is_supported_image?(upload.original_filename)
```

Comment:
Let's also test this as well to make sure images are never added as attachments.